lib/money/money/formatting.rb

```diff
@@ -343,6 +344,15 @@ def regexp_format(formatted, rules, decimal_mark, symbol_value)
     end
   end
 
+  def translate_formatting_rules(rules)
+    begin
+      rules[:symbol] = I18n.t currency.iso_code, :scope => "number.currency.symbol", :raise => true
```

Comment:
This pattern seems pretty expensive in a situation where no translations are used. Can this entire behavior be made optional, as it seems like it's probably not the majority use case?